File: app/src/main/java/com/sktl/mrtask/CardViewActivity.java
package com.sktl.mrtask;

import android.app.Activity;
import android.os.Bundle;
import android.widget.Button;
import android.widget.ImageView;
import android.widget.TextView;

/**
 * Created by USER-PC on 18.02.2018.
 */

public class CardViewActivity extends Activity {

    ImageView processPhoto;
    TextView processName;
    Button button;


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

        setContentView(R.layout.item_view);
        processName = (TextView)findViewById(R.id.process_name);
        button = (Button)findViewById(R.id.process_button);
        processPhoto = (ImageView)findViewById(R.id.process_photo);

        processName.setText("Emma Wilson");
        processPhoto.setImageResource(R.drawable.work_performance256);
    }


}

File: app/src/main/java/com/sktl/mrtask/CardViewActivity.kt
package com.sktl.mrtask

import android.app.Activity
import android.os.Bundle
import android.widget.Button
import android.widget.ImageView
import android.widget.TextView

/**
 * Created by USER-PC on 18.02.2018.
 */

class CardViewActivity : Activity() {

    internal lateinit var processPhoto: ImageView
    internal lateinit var processName: TextView
    internal lateinit var button: Button


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        setContentView(R.layout.item_view)
        processName = findViewById<TextView>(R.id.process_name)
        button = findViewById<Button>(R.id.process_button)
        processPhoto = findViewById<ImageView>(R.id.process_photo)

        processName.text = "Emma Wilson"
        processPhoto.setImageResource(R.drawable.work_performance256)
    }


}
